src/kafka/notifications/src/index.ts

```diff
@@ -2,7 +2,8 @@ import * as fs from "fs";
 import sendEmail from './sendEmail';

```

Comment:
This file uses 4 spaces while sendmail uses 2 spaces for indentation

Reply:
This file has a lot of inner function args that make it seem like it's a 4 space indentation.